adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Http/AdalHttpClient.cs

```diff
@@ -200,7 +200,7 @@ private async Task<T> HandleDeviceAuthChallengeAsync<T>(IHttpWebResponse respons
 
         private static string CheckForExtraQueryParameter(string url)
         {
-            string extraQueryParameter = new PlatformInformation().GetEnvironmentVariable("ExtraQueryParameter");
+            string extraQueryParameter = PlatformProxyFactory.GetPlatformProxy().GetEnvironmentVariable("ExtraQueryParameter");
```

Comment:
>GetPlatformProxy [](start = 62, length = 16)

this thing creates a new object every time. is this really needed? seems to be waste of cpu cycles